docs/reference/migration/migrate_5_1.asciidoc

```diff
@@ -1,6 +1,15 @@
 [[breaking-changes-5.1]]
 == Breaking changes in 5.1
 
+[[breaking_51_scripting_api_changes]]
+[float]
+=== Scripting API changes
+
+==== Stored scripts
+
+Use the parameter 'stored' in place of 'id' when using stored scripts in a query.
```

Comment:
I think it is nice to say something like "Using `id` to refer to a stored script has been deprecated. Use `stored` instead. See <<modules-scripting-stored-scripts,stored scripts>> for examples."

I think the word "deprecated" is important here. Since I can't think off hand how it used to look, maybe a quick inline example. Was it like `"stored": {"id": "foo"}`?

Also use backticks for `stored` and `id`.